$ diff -urN arpack-ng-3.8.0/CMakeLists.txt.orig arpack-ng-3.8.0/CMakeLists.txt
--- arpack-ng-3.8.0/CMakeLists.txt.orig	2022-04-07 18:56:04.575147700 +0200
+++ arpack-ng-3.8.0/CMakeLists.txt	2022-04-07 19:06:38.368200700 +0200
@@ -213,7 +213,6 @@
         set_target_properties(BLAS::BLAS PROPERTIES INTERFACE_LINK_LIBRARIES "${BLAS_LIBRARIES}")
     endif()
 endif()
-get_target_property(BLAS_LIBRARIES BLAS::BLAS INTERFACE_LINK_LIBRARIES) # Get variables from target (*.pc/cmake, msg).
 
 if (MPI)
     if (NOT TARGET MPI::MPI_Fortran) # Search only if not already found by upper CMakeLists.txt
@@ -265,7 +264,6 @@
         set_target_properties(LAPACK::LAPACK PROPERTIES INTERFACE_LINK_LIBRARIES "${LAPACK_LIBRARIES}")
     endif()
 endif()
-get_target_property(LAPACK_LIBRARIES LAPACK::LAPACK INTERFACE_LINK_LIBRARIES) # Get variables from target (*.pc/cmake, msg).
 
 if (MPI)
     set(parpackutil_DIR ${arpack_SOURCE_DIR}/PARPACK/UTIL/)
@@ -696,7 +694,7 @@
 set(ARPACK_PC_LIBS_PRIVATE)
 foreach(lib ${LAPACK_LIBRARIES})
     get_filename_component(libname ${lib} NAME)
-    string(REGEX REPLACE "^lib([^.]+).*$" "-l\\1" libname ${libname})
+    string(REGEX REPLACE "^(lib)?([^.]+).*$" "-l\\2" libname ${libname})
     list(APPEND ARPACK_PC_LIBS_PRIVATE "${libname}")
 endforeach()
 string(REPLACE ";" " " ARPACK_PC_LIBS_PRIVATE "${ARPACK_PC_LIBS_PRIVATE}")
